Need to escape argument with space

I have test functions having space in it's name. If I run it via mirakle, build is failed. I suppose that gradle arguments are sent to remote server well, but some arguments need to be escaped.

I've test on gradle 7.2 / mirakle 1.5.0 / android gradle plugin 7.1.1

Here's my gradle command

```
 ./gradlew testDebugUnitTest --tests "MyUnitTestClass.testFunction haha"
============REMOTE GRADLE ARGUMENTS===============
-Pmirakle.build.on.remote=true
...
--tests
MyUnitTestClass.TestFunction haha   // <-- arguments sent well. but not escpaed?
...
============REMOTE GRADLE ARGUMENTS===============
...
FAILURE: Build failed with an exception.

* What went wrong:
Task 'haha' not found in root project 'MyProject'.
``` 

You can reproduce this by create an unit test and make a  function with space in it's name just like below

```
class MyUnitTestClass {
   @Test
   fun `testFunction haha`() {
     ...
   }
}

```

--- a/plugin/src/main/kotlin/Mirakle.kt
+++ b/plugin/src/main/kotlin/Mirakle.kt
@@ -143,9 +143,9 @@ open class Mirakle : Plugin<Gradle> {
                     args(config.sshArgs)
                     args(
                             config.host,
-                            "${config.remoteFolder}/\"${gradlewRoot.name}\"/gradlew",
+                            "\"${config.remoteFolder}/${gradlewRoot.name}/gradlew\"",
                             "-P$BUILD_ON_REMOTE=true",
-                            "-p ${config.remoteFolder}/\"${gradlewRoot.name}\""
+                            "-p \"${config.remoteFolder}/${gradlewRoot.name}\""
                     )
                     startParamsCopy.copy()
                             .apply {
@@ -179,7 +179,7 @@ open class Mirakle : Plugin<Gradle> {
                 val download = project.task<Exec>("downloadFromRemote") {
                     setCommandLine("rsync")
                     args(
-                            "${config.host}:${config.remoteFolder}/\"${gradlewRoot.name}\"/",
+                            "${config.host}:${config.remoteFolder}/${gradlewRoot.name}/",
                             "./",
                             "--rsh",
                             "ssh ${config.sshArgs.joinToString(separator = " ")}",
@@ -438,8 +438,8 @@ open class MirakleExtension {
 
 fun startParamsToArgs(params: StartParameter) = with(params) {
     emptyList<String>()
-            .plus(taskNames.minus("mirakle"))
-            .plus(excludedTaskNames.map { "--exclude-task $it" })
+            .plus(taskNames.minus("mirakle").map { "\"$it\"" })
+            .plus(excludedTaskNames.map { "--exclude-task \"$it\"" })
             .plus(booleanParamsToOption.map { (param, option) -> if (param(this)) option else null })
             .plus(negativeBooleanParamsToOption.map { (param, option) -> if (!param(this)) option else null })
             .plus(projectProperties.minus(excludedProjectProperties).flatMap { (key, value) -> listOf("--project-prop", "\"$key=$value\"") })

